lib/arrow/shuttle/shape.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Arrow.Shuttle.Shape do
+  @moduledoc "schema for shuttle shapes"
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "shapes" do
+    field :name, :string
+
+    timestamps()
```

Comment:
```suggestion
    timestamps(type: :utc_datetime)
```
and migration update.

Reply:
Since we had a few dependent PRs (https://github.com/mbta/arrow/pull/984 is branched off of this, this PR was branched off another), I moved this into a new PR.